src/pages/abs/[id]/graphics.tsx

```diff
@@ -0,0 +1,95 @@
+import AdsApi, { IADSApiGraphicsParams, IADSApiSearchParams, IDocsEntity, IUserData } from '@api';
+import { GetServerSideProps, NextPage } from 'next';
+import Head from 'next/head';
+import React from 'react';
+import { abstractPageNavDefaultQueryFields } from '@components/AbstractSideNav/model';
+import { AbstractSideNav } from '@components';
+import { normalizeURLParams } from '@utils';
+import { IADSApiGraphicsResponse } from '@api/lib/graphics/types';
+import Link from 'next/link';
+
+interface IGraphicsPageProps {
+  graphics: IADSApiGraphicsResponse;
+  originalDoc: IDocsEntity;
+  error?: string;
+}
+
+const GraphicsPage: NextPage<IGraphicsPageProps> = (props: IGraphicsPageProps) => {
+  const { originalDoc, graphics, error } = props;
+  return (
+    <section className="abstract-page-container">
+      <Head>
+        <title>Graphics | {originalDoc.title}</title>
+      </Head>
+      <AbstractSideNav doc={originalDoc} />
+      <article aria-labelledby="title" className="flex-1 my-8 px-4 py-8 w-full bg-white shadow sm:rounded-lg">
+        {error ? (
+          <div>No Graphics</div>
+        ) : (
+          <>
+            <div className="border-b border-gray-200 sm:pb-0 md:pb-3">
+              <h2 className="prose-xl text-gray-900 font-medium leading-6" id="title">
+                <em>Graphics from</em> <strong>{originalDoc.title}</strong>
+              </h2>
+              <div className="my-2" dangerouslySetInnerHTML={{ __html: graphics.header }}></div>
+            </div>
+            <div className="flex flex-wrap">
+              {graphics.figures.map((figure, index) => {
+                return (
+                  <div
+                    key={index}
+                    className="flex flex-col items-center justify-between m-2 p-2 border-2 border-gray-100 rounded-lg"
+                  >
+                    <Link href={figure.images[0].highres}>
+                      <a target="_blank" rel="noreferrer noopener">
+                        <img src={figure.images[0].thumbnail} className="p-5" alt={figure.figure_label}></img>
```

Comment:
Any reason not to use Next's `<Image/>` tag here?

Reply:
Changed to `<Image>`